### PR TITLE
Readme game instructions

### PR DESCRIPTION
**Pull request by SamroodAli**
Please review our code for milestone 4: Readme Instructions
We have added titles and starting messages to the game.
We have recorded a terminal session playing the game and cast it into an SVG in the readme.
We also wrote an instruction message about the rules of the game as well as getting the game to run.
Finally, we added the author's information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +74,16 @@ Clone repository from github to your local drive by following these steps:
...
 
+![Marijan's GitHub stats](https://github-readme-stats.vercel.app/api?username=marijanbrvar&count_private=true&theme=dark&show_icons=true)
+
+
 👤 SAMROOD ALI
 
 - GitHub: [@SamroodAli](https://github.com/SamroodAli)
 - LinkedIn: [Samrood Ali](https://www.linkedin.com/in/samrood-ali/)
 
+> ![Samrood Ali's GitHub stats](https://github-readme-stats.vercel.app/api?username=SamroodAli&count_private=true&theme=dark&show_icons=true)
```
Comment (pacyL2K19) [inline README.md:85]: **[Optional]** Kindly remove the `>` symbol to keep the same display of contributors in the readme file

**Review by pacyL2K19 — APPROVED** [review]
# Status: Approved :medal_sports: 

Hello **Team** The readme looks professional and no breaks up in the game logic
### Congratulations :tada: 

Before you merge this branch to the `develop` branch kindly consider the optional required change below and go to the next project 

>
### Happy coding :computer:

File: README.md (hunk 1 above)
Comment (SamroodAli) [reply README.md]: Thank you @pacyL2K19 for taking the time to review and approve our code. We have implemented the optional requirement to remove the `>` symbol from the readme. Have a great day and Happy coding!

Comment (pacyL2K19) [reply README.md]: It's an honor, keep it high

**Pull request merged**